brainglobe_atlasapi/list_atlases.py

```diff
@@ -43,11 +45,15 @@ def get_local_atlas_version(atlas_name):
     """
 
     brainglobe_dir = config.get_brainglobe_dir()
-    return [
-        f.name.split("_v")[1]
-        for f in brainglobe_dir.glob(f"*{atlas_name}*")
-        if f.is_dir()
-    ][0]
+    try:
+        return [
+            re.search(r"_v(\d+\.\d+)$", f.name).group(1)
+            for f in brainglobe_dir.glob(f"*{atlas_name}*")
+            if f.is_dir() and re.search(r"_v(\d+\.\d+)$", f.name)
+        ][0]
+    except IndexError:
+        print(f"No atlas found with the name: {atlas_name}")
+        return None
```

Comment:
Might be good to add a very simple test that checks this error is raised if the regex is not matched?

Reply:
Good shout. I was so far down the rabbit hole I forgot about tests!